lld/test/ELF/aarch64-got-relocations-pauth.s

```diff
@@ -0,0 +1,90 @@
+# REQUIRES: aarch64
+
+# RUN: rm -rf %t && split-file %s %t && cd %t
+
+# RUN: llvm-mc -filetype=obj -triple=aarch64 %p/Inputs/shared.s -o a.o
+# RUN: ld.lld -shared a.o -o a.so
+
+#--- ok.s
+# RUN: llvm-mc -filetype=obj -triple=aarch64 ok.s -o ok.o
+
+# RUN: ld.lld ok.o a.so -pie -o ok1
+# RUN: llvm-readelf -r -S -x .got ok1 | FileCheck %s --check-prefix=EXTERNAL
```

Comment:
I'd avoid `EXTERNAL` and `LOCAL` as check prefixes as well since it's unclear what external/local mean in the test context.

Reply:
Thanks for pointing this out - I've changed the output file names but forgot to change the check prefixes previously.

Fixed in 03f9115ae4f3ea45edce60d2aec020bf76d4c2ff and also applied to #113816, thanks!